stix_shifter/stix_transmission/src/modules/qradar/arielapiclient.py

```diff
@@ -1,108 +1,129 @@
-from .RestApiClient import RestApiClient
+from ..utils.RestApiClient import RestApiClient
 import urllib.parse
+import logging
+import sys
+import json
 
 
-# Inherits methods from RestApiClient
-class APIClient(RestApiClient):
+class APIClient():
 
     # API METHODS
 
     # These methods are used to call Ariel's API methods through http requests.
     # Each method makes use of the http methods below to perform the requests.
 
     # This class will encode any data or query parameters which will then be
-    # sent to the call_api() method of its inherited class.
-    def __init__(self, server_ip, auth_token, proxy, cert):
-
+    # sent to the call_api() method of the RestApiClient
+    def __init__(self, connection, configuration):
         # This version of the ariel APIClient is designed to function with
         # version 6.0 of the ariel API.
-        self.endpoint_start = 'ariel/'
-        super(APIClient, self).__init__(server_ip, auth_token, cert, proxy,
-                                        version='8.0')
+
+        print("connection: " + str(connection))
+        self.endpoint_start = 'api/ariel/'
+        headers = dict()
+        hostPort = connection.get('host') + ':' + str(connection.get('port', ''))
+        headers['Version'] = '8.0'
+        headers['Accept'] = 'application/json'
+        auth = configuration.get('auth')
+        if auth != None and auth.get('SEC', None) != None:
+            headers['Sec'] = auth.get('SEC')
+        urlModifierFunction = None
+        proxy = connection.get('proxy')
+        if proxy is not None:
+            proxy_url = proxy.get('url')
+            proxy_auth = proxy.get('auth')
+            if (proxy_url is not None and proxy_auth is not None):
+                headers['proxy'] = proxy_url
+                headers['Proxy-authorization'] = 'Basic ' + proxy_auth
+            if proxy.get('x_forward_proxy', None) is not None:
```

Comment:
I'm confused with naming convention for headers objects..

`x_forward_proxy` vs `X-forward-url` - using different capitalization and symbols, it would be nice to remain consistent

Reply:
Or is it different between proxy vs headers for some reason?

Reply:
Nevermind, this looks fine

Reply:
made all headers in lower-case